src/main/java/com/amazon/opendistroforelasticsearch/security/privileges/DlsFlsEvaluator.java

```diff
@@ -67,7 +68,7 @@ public DlsFlsEvaluator(Settings settings, ThreadPool threadPool) {
     }
 
     public PrivilegesEvaluatorResponse evaluate(final ActionRequest request, final ClusterService clusterService, final IndexNameExpressionResolver resolver, final Resolved requestedResolved, final User user,
-                                                final SecurityRoles securityRoles, final PrivilegesEvaluatorResponse presponse) {
+                                                final SecurityRoles securityRoles, final PrivilegesEvaluatorResponse presponse) throws InvalidIndexNameException {
```

Comment:
Is it necessary to add `InvalidIndexNameException` to `throws`?

Reply:
Removed the `throws` and caught the `RuntimeException` in `OpenDistroSecurityFilter`.